frontend/src/pages/data/SubjectView.tsx

```diff
@@ -1023,7 +1036,7 @@ const SubjectView = observer((props: SubjectViewProps) => {
                 <div className="dz-message needsclick">
                     <i className="h3 text-muted dripicons-cloud-upload"></i>
                     <h5>
-                        Drop C3D or TRC files here to create trials.
+                        "Drop C3D or TRC files here to create trials. You can also drop MOT files, but they must have a TRC or C3D file associated (same name)."
```

Comment:
Remove the quotes around this help message.

Just noticed this on the `dev` frontend deployment.

Reply:
Done!